spec/models/spree/retail/shopify/generate_pos_order_spec.rb

```diff
@@ -12,8 +12,11 @@
 
   let!(:response_mock) { mock_request('orders/450789469', 'json') }
   let(:order_response) { ShopifyAPI::Order.find('450789469') }
+  let(:shopify_transaction) { double('transaction', id: 123) }
```

Comment:
Is there a way to grab this from the order information returned from Shopify's API?